### DISCUSSION TITLE
Add breadcrumb menu to snippet pages

### DISCUSSION THREAD

**Discussion by enzedonline** [wagtail/wagtail#8472] [Ideas]
Would be very useful to have the same breadcrumb menu on snippets that appears at the top of the pages listing/editing interface.

Currently, on saving a snippet, we have to click on the snippet link in the left menu panel or hit history back twice to return to the snippet listing page. 

Something similar to the following:

![image](https://user-images.githubusercontent.com/72310199/166445633-0433ef7f-a6b3-43b8-a220-74819d54dbc5.png)

**Comment by lb-** [comment]
I think this can be created as an enhancement request. I agree this would be good to have.

**Comment by phildexter** [comment]
Nice idea.

@laymonage is working on adding this in at the moment as part of the overarching improvements to Snippets happening for Wagtail 4 😄

**Reply by lb-** [reply]
Converted to issue and assigned to @laymonage

**Reply by lb-** [reply]
https://github.com/wagtail/wagtail/issues/8597

**Reply by enzedonline** [reply]
Great!
